tfgrid3/umbrel/dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    UMBREL_VERSION="release" \
+    UMBREL_REPO="getumbrel/umbrel" \
+    UMBREL_INSTALL_PATH="/umbrel" 
+
+RUN apt update && \
+    apt -y install wget curl vim net-tools iputils-ping openssh-server docker.io \
+    fswatch jq rsync sudo iproute2 git gettext-base python3 gnupg avahi-daemon avahi-discover libnss-mdns nginx ufw
+
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
+    chmod +x /usr/local/bin/docker-compose
+
+RUN mkdir -p ${UMBREL_INSTALL_PATH}; \
+    mkdir -p /etc/zinit;
+COPY ./scripts /scripts
+RUN chmod -R +x /scripts; 
+RUN /bin/bash -c  "/scripts/yq.sh;"
+
+
+RUN version=$(get_umbrel_version); \
+    curl --location "https://api.github.com/repos/${UMBREL_REPO}/tarball/${version}" | \
```

Comment:
in commands depend on the ability to pipe the output of one command into another using the pipe character (|), Docker executes these commands using the `/bin/sh -c` interpreter, which only evaluates the exit code of the last operation in the pipe to determine success.
If you want the command to fail due to an error at any stage in the pipe, set -o pipefail to ensure that an unexpected error prevents the build from inadvertently succeeding.

you can achieve this by Set the SHELL option -o pipefail before RUN with a pipe in
```bash
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```